wfsim/strax_interface.py

```diff
@@ -714,13 +714,20 @@ def compute(self):
 
         chunk = {}
         for data_type in self.provides:
-            if 'nv' in data_type:
+            if 'nv' in data_type and 'nveto' in self.config['targets']:
                 chunk[data_type] = self.chunk(
                     start=self.sim_nv.chunk_time_pre,
                     end=self.sim_nv.chunk_time,
                     data=result_nv[data_type.strip('_nv')],
                     data_type=data_type)
-            else:
+                #If nv is not one of the targets just return an empty chunk
+            if 'nv' in data_type:
+                chunk[data_type] = self.chunk(
+                    start=self.sim.chunk_time_pre,
+                    end=self.sim.chunk_time,
+                    data=np.array([]),
+                    data_type=data_type)
+            elif 'nv' not in data_type:
```

Comment:
I don't understand the logic here. Why not...?
```
if 'nv' in data_type:
    if 'nveto' in self.config['targets']:
        bla
    #If nv is not one of the targets just return an empty chunk
    else:
        blabla
else:
    blablabla
```

The current code would redefine `chunk[data_type]` twice, if nveto is one of the targets, wouldn't it?

Reply:
Fixed here: https://github.com/XENONnT/WFSim/pull/147/commits/322807f2faaeea961d0c40e81454abc846794187